app/Http/Controllers/ExportController.php

```diff
@@ -70,10 +70,12 @@ public function export($table, $personal_key)
                 $data = Company::all();
                 break;
             case 'events':
-                if ($user->can('admin')) {
+                if ($user->can('sysadmin')) {
```

Comment:
This should be 'admin' as a user with the 'board' role may also view all events.